turbo.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "baseBranch": "origin/master",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**", "build/**", "target/**"]
+    },
+    "test": {
+      "dependsOn": ["build"],
+      "outputs": [],
+      "inputs": ["tests/**/*.ts"]
+    },
+    "lint": {
+      "outputs": []
+    },
+
+    "lint:check": {
+      "outputs": []
+    },
+    "deploy": {
+      "dependsOn": ["build", "test", "lint"],
```

Comment:
Does the order matter? Can we move lint forward in the list to fail a build if linting doesn't pass before running all the tests?

Reply:
Order doesn't matter. Deploy will run after all 'build', 'test', 'lint'  have pass.
Deploy can be publishing client library to npm, deploy program to devnet, etc.

Reply:
We can't enforce that linting is successful before running the tests?

Reply:
Yeah, we can. I will update the pipeline.
The order should be lint -> build -> test, right?

Reply:
Yup

Reply:
Updated